src/main/App.ts

```diff
@@ -89,7 +97,7 @@ export class App {
     const restaurantRouter = new RestaurantRouter(restaurantController);
     app.use(Route.RESTAURANTS, restaurantRouter.getRoutes());
 
-    const promotionController = new PromotionController();
+    const promotionController = new PromotionController(nodeGeocoder);
```

Comment:
Oh wait sorry one minor thing, could we pass in the GeocodingService class into PromotionController? I noticed that we're instantiating a geoCodingService inside PromotionController. The slight problem with that is we are no longer doing dependency injection since PromotionController is now saying "I will instantiate a PromotionService" rather than, "I have been injected a PromotionService".

here's an example of how we do it for restaurants
![image](https://user-images.githubusercontent.com/49849754/112742301-5153ec00-8f42-11eb-83e5-05fdd596eb04.png)

Reply:
Oooh yeah sure I can change it, I thought you wanted me to pass in the instance instead of the actual service class earlier but I'll change it back!